src/components/expanded-state/unique-token/UniqueTokenExpandedStateContent.js

```diff
@@ -56,6 +55,28 @@ const UniqueTokenExpandedStateImage = ({ asset }) => {
       asset.image_thumbnail_url;
   const { dimensions: imageDimensions } = useImageMetadata(imageUrl);
 
+  const pixelRatio = PixelRatio.get();
+  const size = Math.ceil((deviceWidth * pixelRatio) / 100) * 100;
+  const url = useMemo(() => {
+    if (
+      asset.image_url?.startsWith?.('https://lh3.googleusercontent.com/') &&
```

Comment:
You should move the url to a const at the top of the file, since it's being reused